README.md

```diff
@@ -69,6 +69,39 @@ The remaining tests can be built by enabling the `AVIF_BUILD_TESTS` and
 `AVIF_ENABLE_GTEST` CMake options. They require GoogleTest to be built locally
 with ext/googletest.cmd or installed on the system.
 
+### Command Lines
+
+The following instructions can be used to build the libavif library and the
+`avifenc` and `avifdec` tools.
+
+#### Build using installed dependencies
+
+To link against the already installed `aom`, `libjpeg` and `libpng` dependency
```

Comment:
I suggest also adding the libsharpyuv-dev package: https://packages.debian.org/search?keywords=libsharpyuv-dev

I wrote https://github.com/AOMediaCodec/libavif/pull/1937 to add libsharpyuv to the "Build everything from scratch" section.